gl_webgl.go

```diff
@@ -1418,6 +1418,15 @@ func (c *Context) Viewport(x, y, width, height int) {
 	c.Call("viewport", x, y, width, height)
 }
 
+func (c *Context) GetViewport() [4]int32 {
+	var params [4]int32
+	vp := c.Call("getParameter", c.VIEWPORT)
+	for i := 0; i < 4; i++ {
+		params[i] = vp.Index(i).Int()
```

Comment:
Do you not have to cast this? That's the only difference between yours and what I did :P

Reply:
Yeah, I get "cannot use type int as type int32" when I try to build it

Reply:
I guess I have to setup a build environment for web and mobile ^^